keps/sig-storage/2268-non-graceful-shutdown/kep.yaml

```diff
@@ -0,0 +1,40 @@
+title: non graceful shutdown
+kep-number: 2268
+authors:
+  - "@yastij"
+  - "@xing-yang"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: 2019-06-26
+reviewers:
+  - "@andrewsykim"
+  - "@jingxu97"
+  - "@jsafrane"
+approvers:
+  - "@msau42"
+  - "@thockin"
+prr-approvers:
```

Comment:
I think prr-approvers got moved to the separate file so this is not needed here anymore

Reply:
Removed.